### ISSUE TITLE
Problems with ParaviewWeb!!!

### ISSUE THREAD

**Issue by chris2009** [Kitware/paraviewweb#131]
HI my friends, I have tried to work with paraview on the web I have followed all the steps from the following link: http://paraviewweb.kitware.com/index.html#!/guide/quick_start;  but I have serious problems with that. First I have compiled by myself the paraview and then I have downloaded the file from this link http://www.paraview.org/download/ and then I putted these comands :
$ tar xvzf ParaView-4.1.0-Linux-64bit-glibc-2.3.6.tar.gz
$ cd ParaView-4.1.0-Linux-64bit
**$ ./bin/pvpython lib/paraview-4.1/site-packages/paraview/web/pv_web_visualizer.py  \
            --content ./share/paraview-4.1/www                                     \
            --data-dir /path-to-share/                                             \
            --port 8080 &
-
=> Then, open a browser to the following URL http://localhost:8080/apps/Visualizer**

Finally the result is that:

root@nesimo-VirtualBox:~/Escritorio/Project/build# ./bin/pvpython lib/site-packages/paraview/web/pv_web_visualizer.py --content ./home/n-esimo/Escritorio/Project/build/www --port 8080 --data-dir ../../home/n-esimo/Escritorio/ParaView-5.0.0/.ExternalData/MD
2016-05-31 14:12:55-0500 [-] Log opened.
pci id for fd 4: 80ee:beef, driver (null)
libGL error: core dri or dri2 extension not found
libGL error: failed to load driver: vboxvideo
2016-05-31 14:12:55-0500 [-] Site starting on 8080
2016-05-31 14:12:55-0500 [-] Starting factory <twisted.web.server.Site instance at 0x7f4e19ffe9e0>
2016-05-31 14:13:14-0500 [HTTPChannel,0,127.0.0.1] "127.0.0.1" - - [31/May/2016:19:13:14 +0000] "GET /apps/Visualizer HTTP/1.1" 404 145 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36"

And when I putted http://localhost:8080/apps/Visualizer, the page show the following: No Such Resource File not found.

Please May you help me with this trouble, thanks very much.

**Comment by jourdain** [comment]
Never got that error message.

But here is what I can tell:
- You are running ParaViewWeb within a virtual box which seems to have graphic driver issue which is a problem in the way you run ParaView.
- It seems some of the path provided on the command line are wrong? ./home ?

Since you are using a virtual box, you can look into our vagrant deployment:
http://kitware.github.io/paraviewweb/docs/guides/vagrant.html

And if you want to try with your current setup, provide valid absolute path and add the following argument to use your CPU instead of your GPU for doing the rendering.

--mesa-llvm

But make sure you use the binary from our download website (5.0.1) as that argument will ask paraview to use the OSMesa library that was build along ParaView which won't be the case for your local build.

**Comment by mathturtle** [comment]
AFAIK, running ParaView 5.X inside VirtualBox **can't** work unless you use the new Mesa options since ParaView requires a graphics card/driver that has fairly substantial OpenGL support.   And even before the switch to newer OpenGL, ParaView and VBox's drivers didn't really work all that well together.

**Comment by chris2009** [comment]
I'll try to solve my problem with both advices, and I'll show the solution, if I found it, here.
Thank you very much for your responses, my friends.

**Comment by chris2009** [comment]
My friends I want to thank you for your advices and for your time, I found the solution!.

 - First, you are rigth when you said me that ParaView and VBox's drivers didn't really work all that well         together, so I installed ubuntu16 and I started to work there.
- I compiled paraview and for paraviewweb you are right, I was wrong in the path, this was the correct path: 
./bin/pvpython lib/site-packages/paraview/web/pv_web_visualizer.py --content www --port 8080 --data-dir Data/ParaView-5.0.0.

Now I am learning how to use this wonderful tool.

Best regards!!!!
